integration-tests/kogito/invoked/root/rpkgtests/pom.xml

```diff
@@ -17,7 +17,7 @@
         <dependency>
             <groupId>org.kie.kogito</groupId>
             <artifactId>kogito-quarkus-integration-test</artifactId>
-            <version>0.15.0</version>
+            <version>1.0.0.Final</version>
```

Comment:
Congrats! :)

Reply:
wait till it builds though :D I am having quite a few issues locally

Reply:
nice, I am bit a confused, I ran locally and now it is green for me @gsmet /o\

Reply:
it's a JDK-8 codegen related issue that I thought we had sorted out -- unfortunately it was occurring in two different places https://github.com/kiegroup/kogito-runtimes/pull/905